react-sidebar/index.d.ts

```diff
@@ -0,0 +1,35 @@
+// Type definitions for react-sidebar 2.2
+// Project: https://github.com/balloob/react-sidebar#readme
+// Definitions by: Jeroen Vervaeke <https://github.com/jeroenvervaeke>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+import * as React from "react";
```

Comment:
You can use `import { ReactNode, ReactElement } from "react"` rather than having a separate line for each import.